src/patternfly/components/Page/page.scss

```diff
@@ -204,6 +215,10 @@
   overflow: hidden;
   background-color: var(--pf-c-page__header-nav--BackgroundColor);
 
+  > * {
```

Comment:
is there a reason you're using `*` instead of `.pf-c-nav`?

Reply:
I was thinking for flexibility but I don't think that's necessary here so good point!